doc/io_streams.rdoc

```diff
@@ -368,13 +406,21 @@ which is the non-negative integer line number
 in the stream where the next read will occur.
 
 The line number is the number of lines read by certain line-oriented methods
-(IO.foreach, IO#each_line, IO#gets, IO#readline, and IO#readlines)
+({#foreach}[https://docs.ruby-lang.org/en/master/IO.html#method-c-foreach],
```

Comment:
I think `foreach` is a class method, so I think the convention is to use `::foreach`.

Reply:
Done.  I'll add something to the guide about this.  We've been using `Foo.bar` when the class name is prefixed but `::bar` when not.